### PR TITLE
feat(docs):[TRI-1604] Add concept for EDR caching and reuse

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

File: docs/concept/EDR-Token-Cache-and-reuse.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# [Concept] EDC EDR Token negotiation and usage
```
Comment (mkanal) [inline docs/concept/EDR-Token-Cache-and-reuse.md:1]: Please add the JIRA-ID as a reference in filename and Please add a meta header section containing the information author and JIRA-ID , Creation date,   Due Date, author Confidentiality level:

File: docs/concept/EDR-Token-Cache-and-reuse.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    EdcDataPlaneClient ->> EdcSubmodelClient: Return data
+```
+
+## New flow - reuse token
+
+To increase performance for assets with the same EDC contract, IRS should be able to reuse an existing EDR token.
+
+To make this possible, two things have to be adjusted.
+
+1. IRS has to check first, if an EDR Token for the requested EDC Asset ID is available in EDR-Storage
```
Comment (mkanal) [inline docs/concept/EDR-Token-Cache-and-reuse.md:34]:
```suggestion
1. IRS has to check first, if an EDR Token for the requested EDC Asset ID is available in EndpointDataReferenceStorage (EDR-Storage)
```

File: docs/concept/EDR-Token-Cache-and-reuse.md (hunk 1 above)
Comment (mkanal) [inline docs/concept/EDR-Token-Cache-and-reuse.md:1]:
```suggestion
# Glossar
| Abbreviation| Name | 
| EDR-Storage |EndpointDataReferenceStorage| 
| EDR|EndpointDataReference| 


# [Concept] EDC EDR Token negotiation and usage
```

File: docs/concept/EDR-Token-Cache-and-reuse.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+### Case: Token already present in EDR-Storage
+
+```mermaid
+flowchart LR
+    A[EDR Token] --> B[extract authCode]
+    B --> C[decode JWT]
+    C --> D[get expiry date 'exp']
+    D --> E{isExpired}
```
Comment (mkanal) [inline docs/concept/EDR-Token-Cache-and-reuse.md:73]: Add case If expiry date is not available.
Add case if expiry date is not valid ?

**Review by mkanal — CHANGES_REQUESTED** [review]

File: docs/concept/EDR-Token-Cache-and-reuse.md (hunk 1 above)
Comment (ds-jhartmann) [reply docs/concept/EDR-Token-Cache-and-reuse.md]: Added only the Jira ID, as discussed.

File: docs/concept/EDR-Token-Cache-and-reuse.md (hunk 2 above)
Comment (ds-jhartmann) [reply docs/concept/EDR-Token-Cache-and-reuse.md:41]: Added this to one section above

File: docs/concept/EDR-Token-Cache-and-reuse.md (hunk 1 above)
Comment (ds-jhartmann) [reply docs/concept/EDR-Token-Cache-and-reuse.md]: Done with correct format

File: docs/concept/EDR-Token-Cache-and-reuse.md (hunk 3 above)
Comment (ds-jhartmann) [reply docs/concept/EDR-Token-Cache-and-reuse.md]: done

**Review by mkanal — APPROVED** [review]

**Review by jzbmw — APPROVED** [review]
looks reasonable. 👍

**Comment by jzbmw** [comment]
I would like to discuss the dependencies to policies.

**Pull request merged**